utils-zookeeper/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,9 @@
+# Root logger option
+log4j.rootLogger=INFO, stdout
+#log4j.rootLogger=DEBUG, stdout
```

Comment:
nit: rm

Reply:
We can remove it, but once we start to debug something there, we will add it again, and again add comments like this to PRs. I prefer to let it be there, so that it can be easily uncommented if needed.